README.md

```diff
@@ -35,8 +40,10 @@ or apt to install these components if they're self-hosted.
 
 ### What splunk affects
 
-* Installs the Splunk/Forwarder package and manages their config files. It does not purge them by default.
-* The module will set up both Splunk and Splunkforwarder to run as the 'root' user on POSIX platforms.
+* Installs the Splunk/Forwarder package and manages their config files. It does
+  not purge them by default.
+* The module will set up both Splunk and Splunkforwarder to run as the 'root'
+  user on POSIX platforms.
```

Comment:
I disagree, because I changed this for almost every module and this is more readable in text editors like VIM.

@bastelfreak What do you think?